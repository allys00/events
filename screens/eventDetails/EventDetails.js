import React, { Component } from 'react';
import { connect } from 'react-redux';
import { View } from 'react-native';
import Text from '../../components/Text';
import ButtonWrapper from '../../components/Button';
import { navigate } from '../../utils/NavigationService';

class EventDetails extends Component {

  // constructor(props) {
  //   super(props)
  //   setNavigator(props.navigation)
  // }

  static navigationOptions = {
    title: 'Detalhes do evento',
  };

  render() {
    return (
      <View>
        <Text>Detalhes</Text>
      </View>
    )
  }
}

const mapStateToProps = ({ login }) => ({ login })

export default connect(mapStateToProps, {})(EventDetails);
