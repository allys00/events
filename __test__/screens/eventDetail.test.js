import React from 'react';
import renderer from 'react-test-renderer';
import EventDetails from '../../screens/eventDetails/EventDetails';
import Store from '../../config/ReduxStore'
import { Provider } from 'react-redux';

describe('Testing events details', () => {
  it('renders correctly', () => {
    const event = {
      title: 'Event',
      image: 'https://s3-us-west-2.amazonaws.com/agendaedu-dev/schools/c5c1a933-cdef-4c9b-8a87-490f25c2538d/events/5380/attachments/1550866911-$1-original-poster-agendakids.jpeg',
      startAt: '2019-07-22T19:01:33.476Z',
      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque pretium nulla non arcu aliquam rhoncus eu sed leo. Aenean cursus nibh sit amet fringilla sodales. Vestibulum faucibus venenatis tempor. Morbi placerat ac massa id ultricies.'
    }
    const tree = renderer
      .create(
        <Provider store={Store}>
          <EventDetails  {...event} />
        </Provider>
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});