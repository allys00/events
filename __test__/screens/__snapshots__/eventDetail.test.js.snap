// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing events details renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#733DBE16",
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "paddingBottom": 0,
          "paddingLeft": 24,
          "paddingRight": 24,
          "paddingTop": 24,
        },
        Object {
          "backgroundColor": "#ffffff",
          "borderTopLeftRadius": 40,
          "borderTopRightRadius": 40,
          "height": 889.3333333333333,
          "marginTop": -20,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexDirection": "row",
          "marginBottom": 20,
          "padding": 10,
          "width": "100%",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#733DBE16",
            "borderRadius": 5,
            "display": "flex",
            "height": 60,
            "justifyContent": "center",
            "width": 60,
          }
        }
      >
        <Text
          color="#333"
          fontSize={22}
          marginBottom={10}
          style={
            Array [
              Object {
                "color": "#333",
                "fontFamily": "System",
                "fontSize": 22,
                "fontWeight": "bold",
                "marginBottom": 10,
              },
              Object {
                "color": "#733DBE",
                "marginBottom": 0,
              },
            ]
          }
          weight="bold"
        >
          
        </Text>
        <Text
          color="#333"
          fontSize="14px"
          marginBottom={10}
          style={
            Array [
              Object {
                "color": "#333",
                "fontFamily": "System",
                "fontSize": 14,
                "fontWeight": "normal",
                "marginBottom": 10,
              },
              Object {
                "color": "#733DBE",
                "marginBottom": 0,
              },
            ]
          }
          weight="normal"
        >
          
        </Text>
      </View>
      <View
        style={
          Object {
            "marginLeft": 15,
          }
        }
      >
        <Text
          color="#333"
          fontSize={22}
          marginBottom={10}
          style={
            Array [
              Object {
                "color": "#333",
                "fontFamily": "System",
                "fontSize": 22,
                "fontWeight": "bold",
                "marginBottom": 10,
              },
            ]
          }
          weight="bold"
        />
        <Text
          color="#666"
          fontSize="14px"
          marginBottom={10}
          style={
            Array [
              Object {
                "color": "#666",
                "fontFamily": "System",
                "fontSize": 14,
                "fontWeight": "normal",
                "marginBottom": 10,
              },
            ]
          }
          weight="normal"
        >
          
        </Text>
      </View>
    </View>
    <Text
      color="#666666"
      fontSize={16}
      marginBottom={10}
      style={
        Array [
          Object {
            "color": "#666666",
            "fontFamily": "System",
            "fontSize": 16,
            "fontWeight": "normal",
            "marginBottom": 10,
          },
        ]
      }
      weight="normal"
    />
  </View>
</View>
`;
